cmd/cluster-network-operator/main.go

```diff
@@ -80,7 +78,7 @@ func main() {
 	}
 
 	// become leader
-	err = leader.Become(context.TODO(), LOCK_NAME)
+	err = Become(context.TODO(), LOCK_NAME)
```

Comment:
now the naming is confusing; the function should be called `BecomeLeader()` or something like that.

Reply:
ack